abis_mapping/templates/survey_occurrence_data/schema.json

```diff
@@ -542,8 +542,8 @@
     {
       "name": "acceptedNameUsage",
       "title": "Accepted Name Usage",
-      "description": "The full name, with authorship and date information if known, of the currently valid (zoological) or accepted (botanical) taxon.",
```

Comment:
I think this is suppose to be the description for all of them, not the `Calaendia excelsa Hopper & A.P.Br.`